### DISCUSSION TITLE
Sponsoring feature development to gotenberg (ghostscript support)

### DISCUSSION THREAD

**Discussion by venkatd** [gotenberg/gotenberg#853] [Q&A]
We would like a way to convert images to PDFs and ghostscript seems like the best route for this.
I saw this fork but it's about 2 years old: https://github.com/Vrex123/gotenberg-ghostscript
We are happy to pay someone to get something merged into the official gotenberg package.

Wanted to check what our best options are!

**Comment by gulien** [comment]
Hello @venkatd,

The [licensing](https://www.ghostscript.com/licensing/) of Ghostscript does not allow it to be merged into Gotenberg.

**Reply by venkatd** [reply]
Ok thanks for clearing that up... that's unfortunate! We managed to find a workaround.

The general question is still relevant.

Is a recommended way to sponsor improvements to the project if we ever run into issues or desire some improvements?

Cheers!

**Reply by gulien** [reply]
Hey @venkatd,

Thanks for your interest! I might setup a GitHub sponsor page for Gotenberg in a near future 👍
